src/settings.ts

```diff
@@ -0,0 +1,38 @@
+
+// here we define all settings that have an effect on how the pareto chart will be rendered and look like, 
+
+
+// for example: tick stroke width, tick length, etc
+export interface Settings {
+    svg: undefined;
+    clearMarking?(): void;
+    windowSize: { width: number; height: number };
+    style: {
+        ticks: {stroke: string};
+        label: { size: number; weight: string; style: string; color: string; fontFamily: string };
+        lines: {weight: string, color: string};
+        //font: { size: number; weight: string; style: string; color: string; fontFamily: string };
+        marking: { color: string };
+        background: { color: string };
+    };
+}
+
+/**
+ * @param setting Setting data structure
+ */
+
+
+export function renderSettings(setting: Settings) {
```

Comment:
I think rendersettings could be useful for defining the windowsize or something that is more general. I think it is better to keep it this way and then remove it if it becomes redundant.

Reply:
I am not sure it would work that way, then you'd have to define the behaviour and probably also rename it, since it would not be readable. but perhaps I'm missing something. I thought it was redundant, but I approved the PR anyways it can stay commented perhaps?  in any case you can decide and merge since it's no longer in review.